falcon/response.py

```diff
@@ -185,23 +188,28 @@ def media(self, obj):
     def __repr__(self):
         return '<%s: %s>' % (self.__class__.__name__, self.status)
 
-    def set_stream(self, stream, stream_len):
-        """Convenience method for setting both `stream` and `stream_len`.
+    def set_stream(self, stream, content_length):
+        """Convenience method for setting both `stream` and `content_length`.
+
+        Args:
```

Comment:
Would you mind moving this Args section to the bottom of the docstring, below the prose sections?